gen/gen.go

```diff
@@ -217,8 +222,8 @@ func (g *Gen) writeGoDoc(output io.Writer, swagger *spec.Swagger) error {
 }
 
 var packageTemplate = `// GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
-// This file was generated by swaggo/swag at
-// {{ .Timestamp }}
+// This file was generated by swaggo/swag{{ if .GeneratedTime }} at
+// {{ .Timestamp }}{{ end }}
```

Comment:
Already have `Timestamp` filed why need another `GeneratedTime` to generate? 😕

Reply:
You mean I should change the name or remove it? rename means changing to `EnableGeneratedTime` or etc, and remove means replacing `if .GeneratedTime` 
 with `if not .Timestamp.IsZero()`

For removing, I think `if .GeneratedTime` would be more explicit.

Reply:
I misunderstand your point. LGTM.